text/0000-intro-video-script.md

```diff
@@ -0,0 +1,73 @@
+# Meta
+[meta]: #meta
+- Name: Intro video script
+- Start Date: 2021-05-18
+- Author(s): yaelharel
+- RFC Pull Request: (leave blank)
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+- Supersedes: N/A
+
+# Summary
+[summary]: #summary
+
+Improving documentation is a project priority for 2021.
+This RFC is an attempt to create an easy and clear starting point for new users and contributors.
+
+# Definitions
+[definitions]: #definitions
+
+N/A
+
+# Motivation
+[motivation]: #motivation
+
+It is currently relatively challenging for new users and contributors to understand what buildpacks are and what is the purpose of our project.
+It can be very useful to have a ~1 minute introductory video to provide high-level answers to these questions.
+We can post this video on our website and share it on social media.
+Hopefully this will help familiarize more people with our project.
+
+
+# What it is
+[what-it-is]: #what-it-is
+
+### The script
+
+As an application developer, you should focus on what you do best - writing code, without having to worry about image security, optimizing container images, or your container build strategy. How much time have you spent struggling to wrangle yet another Dockerfile? Copying and pasting random Dockerfile snippets into every project? Buildpacks can help you with just that! They are a better approach to building container images for applications.
```

Comment:
Do we care about addressing another target audience or this is just a video for app developers?

Reply:
The purpose of this video is to make a clear and short introduction to what buildpacks are.
I think that the easiest way to address it is from the app developers perspective.

Reply:
My dream: one day we have a separate video for platform operators :)

Reply:
@natalieparellano, it will be great!